specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2022-09-01-preview/apimanagement.json

```diff
@@ -613,6 +613,47 @@
       "default": "xml",
       "x-ms-parameter-location": "method"
     },
+    "PolicyRestrictionIdParameter": {
+      "name": "policyRestrictionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Policy restrictions for a product",
+      "minLength": 1,
+      "maxLength": 80,
+      "x-ms-parameter-location": "method"
+    },
+    "PolicyRestrictionComplianceState": {
```

Comment:
Where this enum will be used? I mean are you sure that this is a parameter which will be passed in header and not as a query parameter?